src/components/tables/DataTable.vue

```diff
@@ -116,7 +116,7 @@ export default {
       }
 
       if (dataCell.type && dataCell.type === 'dynamic') {
-        return $(node).find('p').text();
+        return parseFloat($(node).find('p').text().replace(/,/g, ''));
```

Comment:
Where does the `,` come from initially? What was responsible for adding the `,` in the first place?

Reply:
> Where does the `,` come from initially? What was responsible for adding the `,` in the first place?

@tonytlwu 
![image](https://user-images.githubusercontent.com/86256215/152321373-35dd6314-3063-45e3-a6de-6402df33ffb5.png)

Reply:
Technically the solution you've given will work because we're **always** localising it to `en`, which uses `,` as the large number divider.

However, I think we can do better and allow a more flexible future (if we choose to localise the number format) without needing to change this part of the code.

If you add `data-value` to the cell and assign the cell value to it (without any localisation or formatting), then you can change this line in DataTable.vue to:

```suggestion
        return node.dataset.value;;
```

This way:

1. The solution will work with all localisation formats
2. It will perform faster because it won't need to go through the `$(node).find('p')` process
3. We're avoiding transforming number to text and then to number again, which, under the wrong scenarios, could be quite troublesome, e.g.

![image](https://user-images.githubusercontent.com/290733/152348519-4eaae1ca-421c-4a47-a48c-af688ed4aa68.png)